blogPosts/en/blog/governance/Maker-Governance-Review-April-2021.mdx

```diff
@@ -0,0 +1,147 @@
+- [UNI-V2-GUSD-DAI (Uniswap V2 GUSD-DAI LP Token](https://vote.makerdao.com/polling/QmXWGE8y))
+- [1INCH (1inch Liquidity Protocol)](https://vote.makerdao.com/polling/QmVcvnGc)
+- [rETH (Rocket Pool Staked ETH) ](https://vote.makerdao.com/polling/QmV4YgXs)
+- [SLP-DAI-USDC (Sushiswap DAI-USDC LP Token) ](https://vote.makerdao.com/polling/QmZpXu3K)
+- [UNI-V2-DAI-PAX (Uniswap V2 DAI-PAX LP Token) ](https://vote.makerdao.com/polling/QmcMSMeJ)
+
+Check the progress of all proposed collateral types in the [Collateral Status Index.](https://forum.makerdao.com/t/collateral-status-index/2231)
+
+# What’s Ahead?
+
+May is here, and there are some things to watch out for. Formal submissions for the May cycle are wrapping up tomorrow, May 5th.
+
+Two more Core Units! Both the [Strategic Marcomms Core Unit](https://forum.makerdao.com/t/mip39c2-sp9-strategic-marcomms-core-unit/7189) and the [Sustainable Ecosystem Scaling Core Unit](https://forum.makerdao.com/t/mip39c2-sp10-adding-sustainable-ecosystem-scaling-core-unit/7368/7) have entered into the May governance cycle.
+
+LongForWisdom and the GovAlpha team propose [changes to the Monthly governance Cycle](https://forum.makerdao.com/t/mip51-monthly-governance-cycle/7366) to streamline the procedure, aiming “to reduce the cost of participation for governance by reducing the number of votes that take place for each monthly cycle.” In addition, [an amendment to clean up MIP0](https://forum.makerdao.com/t/mip4c2-sp14-mip0-amendments/7260) is also in the pipeline for May.
+
+MIP50, [the Direct Deposit Module](https://forum.makerdao.com/t/mip50-direct-deposit-module/7356), is also being entered into formal submission. The Direct Deposit Module interfaces with third-party lending protocols to directly supply Dai to their markets. Maker Governance is able to pick a target variable borrow rate, and the module will automatically deposit/remove liquidity to ensure that the target rate is hit. This will be yet another technical MIP which, if passed, will be added to the Protocol Engineering teams growing backlog alongside Dai Flash Mint Module, Governance Contract Upgrade, and others.
+
+Finally, MIP49, [Staking Rewards](https://forum.makerdao.com/t/mip49-staking-rewards/6331/72), is being entered into formal submission. MIP49 covers the creation of a Staking Rewards process for MKR token holders, designed to distribute DAI as a percentage of protocol profits. MIP49 would entitle MKR holders to a share of this percentage based on the weight of their MKR staked in the Governance Contract. Should this proposal be accepted by the community, the plan would be to use @hexonaut’s DssGovRewards Code to implement. The MIP also mentions its intentionality behind not setting a particular path for the distributions to allow for different implementations in the future and some flexibility around future Governance improvements.
+
+![](https://i.imgur.com/N8hRvoH.png)
```

Comment:
@Davidutro you can use imgur links as long as the file extension is there. 

The path would just be `static/images/whataver-file-name.png/jpg` and then thus syntax would be: 
 `![](/images/whataver-file-name.png)`